packages/core/src/repository/data-source/sql/raw-sql.data-source.ts

```diff
@@ -15,7 +15,7 @@ import {
     IdQuery,
     IdsQuery,
     PaginationOffsetLimitQuery,
-    Query,
+    Query, BaseColumnDeletedAt,
```

Comment:
One import per line?

Reply:
Does it matter?

Reply:
Just a matter of style, it's not relevant.